pkg/kubelet/container/sync_result.go

```diff
@@ -68,6 +70,8 @@ const (
 	ConfigPodSandbox SyncAction = "ConfigPodSandbox"
 	// KillPodSandbox action
 	KillPodSandbox SyncAction = "KillPodSandbox"
+	// ResizePod action
```

Comment:
nit: add a little more detail here
```suggestion
	// ResizePod action is included whenever any containers in the pod are resized without restart
```

Reply:
done